backend/src/controllers/admin/index.js

```diff
@@ -29,4 +29,19 @@ const addLesson = asyncHandler(async (req, res) => {
   res.status(200).json({ data: new_lesson });
 });
 
-module.exports = { addLesson };
+const viewLessons = asyncHandler(async (req, res) => {
+  //set Offset
+  let offset = req.params.offset || 0;
+  offset = Number(offset);
+  let limit = 10;
```

Comment:
You can declare `10` as constant variable under `utils/constant.js` file since this will be used in the future